content/en/docs/v3.5/demo.md

```diff
@@ -245,18 +245,16 @@ etcdctl --endpoints=$ENDPOINTS elect one p2
 
 Specify the initial cluster configuration for each machine:
 
-![10_etcdctl_endpoint_2016050501](https://storage.googleapis.com/etcd/demo/10_etcdctl_endpoint_2016050501.gif)
-
 ```shell
 etcdctl --write-out=table --endpoints=$ENDPOINTS endpoint status
 
-+------------------+------------------+---------+---------+-----------+-----------+------------+
-|     ENDPOINT     |        ID        | VERSION | DB SIZE | IS LEADER | RAFT TERM | RAFT INDEX |
-+------------------+------------------+---------+---------+-----------+-----------+------------+
-| 10.240.0.17:2379 | 4917a7ab173fabe7 | 3.0.0   | 45 kB   | true      |         4 |      16726 |
-| 10.240.0.18:2379 | 59796ba9cd1bcd72 | 3.0.0   | 45 kB   | false     |         4 |      16726 |
-| 10.240.0.19:2379 | 94df724b66343e6c | 3.0.0   | 45 kB   | false     |         4 |      16726 |
-+------------------+------------------+---------+---------+-----------+-----------+------------+
++------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+|    ENDPOINT      |        ID        | VERSION | DB SIZE | IS LEADER | IS LEARNER | RAFT TERM | RAFT INDEX | RAFT APPLIED INDEX | ERRORS |
++------------------+------------------+---------+---------+-----------+------------+-----------+------------+--------------------+--------+
+| 10.240.0.17:2379 | 4917a7ab173fabe7 |  3.4.15 |   45 kB |      true |      false |         4 |      16726 |              16726 |        |
```

Comment:
As commented earlier, the version should reflect 3.5 (e.g. 3.5.0) here. Thanks!

Reply:
Fine.